get_box_mesh.py

```diff
@@ -0,0 +1,14 @@
+def get_box_mesh(dim,a,b,n, t=None, periodic=None):
+    if periodic is None:
+        periodic = (False)*dim
```

Comment:
```suggestion
        periodic = (False,)*dim
```
(Otherwise I think it doesn't become a tuple.)